src/scripts/main.js

```diff
@@ -1 +1,69 @@
 'use strict';
+
+function handleNotification(divMessage, isSuccess) {
+  const div = document.createElement('div');
+
+  div.textContent = divMessage;
+  div.className = isSuccess ? 'success' : 'error';
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
```

Comment:
The first promise has a potential issue where it can both resolve and reject. If a click event occurs before the timeout, it will resolve, but the timeout will still trigger a rejection after 3 seconds. Consider clearing the timeout upon resolution to prevent this.